subsys/bluetooth/services/lbs.c

```diff
@@ -46,8 +46,20 @@ static ssize_t write_led(struct bt_conn *conn,
 	LOG_DBG("Attribute write, handle: %u, conn: %p", attr->handle,
 		(void *)conn);
 
+	if (len != 1U) {
```

Comment:
I didn't noticed it in the first round, but since we are adding validation of len, we should also do it for offset.
```
if (len != 0) {
   return BT_GATT_ERR(BT_ATT_ERR_INVALID_OFFSET);
}
```

Reply:
Offset checking has been added.